keyboards/handwired/illera16/rules.mk

```diff
@@ -0,0 +1,4 @@
+# This file intentionally left blank
+RGBLIGHT_ENABLE = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
Encoder map should only be enabled at the keymap level. 
```suggestion
```

Reply:
done